### DISCUSSION TITLE
Russian Translation

### DISCUSSION THREAD

**Discussion by Alex-Sokolov** [vuejs-translations/guidelines#12] [General]
Ah... here we go again...

**Comment by dudintv** [comment]
Кто-то уже взялся? 
Я готов поучаствовать, но не хочется делать лишнего :)

**Reply by Alex-Sokolov** [reply]
Пока ещё непонятно за что браться. Гайдлайнов ещё нет, реп тоже. Да и пока не очень понятно что из текущей русской документации к третьей версии переносимо, а что нет. Просто ждём

**Comment by samenick** [comment]
Добрый день. Могу помочь когда появятся гайдлайны

**Comment by Hkr32** [comment]
Готов помочь, если возьмете)

**Comment by Delari** [comment]
Так же, чем смогу - помогу

**Comment by d-darwin** [comment]
Буду рад поучаствовать.

**Comment by Askay35** [comment]
Как начнете дайте знать - буду рад посодействовать o/

**Comment by farid-temuri-lofty** [comment]
Если не проявлю активность после появления гайдлайна - тегайте.

**Comment by frontend-rogatnev** [comment]
Пните, когда можно будет помочь

**Comment by d-darwin** [comment]
Испанские коллеги просто форкнули репу и поехали
https://github.com/vuejs-translations/guidelines/discussions/3#discussioncomment-2601591
https://vue3-spanish.netlify.app/
Может тоже возьмем все в свои руки?

**Reply by Alex-Sokolov** [reply]
Проблема не в форке, проблема в поддержке :) уже давно от репы был сделан форк и перевод сделан и прекрасно поживает (https://v3.ru.vuejs.org/ru/). Но никто не предполагал что ещё раз поменяется всё

**Comment by Roccowen** [comment]
Буду рад поучавствовать!

**Comment by MaximSavin** [comment]
Есть какие-то предпосылки к началу перевода?

_Буду рад помочь!_

**Reply by nzuyanov** [reply]
Готов поучаствовать

**Comment by baraboza** [comment]
Почему нельзя просто взять и начать перевод существующей документации?

**Reply by MaximSavin** [reply]
https://github.com/vuejs-translations/guidelines/discussions/12#discussioncomment-2727547
_вот тут ответили уже..._
Проблема в поддержке, очень желательно переводить синхронно с разработчиками, чтобы не впустую. И чтобы на основном сайте Vue3 появился раздел с русской документацией.

**Comment by Biseamon** [comment]
I am in!

**Comment by dsvitnev** [comment]
@Alex-Sokolov тегните пожалуйста меня в дискуссии как появятся новые гайдлайны 🙏🏻

**Comment by samenick** [comment]
Добрый день.

Судя по странице https://vuejs.org/translations/ идет процесс перевода на упрощенный китайский и японский языки. Гайдов нет, и, судя по всему, не будет. Я полагаю, что можно пытаться наладить контакт с мейнтейнерами и пилить русскую локализацию

**Reply by XamzatJR** [reply]
Теперь есть

**Comment by 3lement3** [comment]
Всем привет.
Буду рад поучаствовать в переводе, если возьмёте. :upside_down_face:

**Comment by Alex-Sokolov** [comment]
Да, гайдлайны появились :) наверное нет смысла тегать каждого лично, кто заметит приходите в репо и будем там организовываться

Форк со старой версией документации отвязал
Новый форк: https://github.com/translation-gang/docs-ru
PR создавать в ветку main
В README описал как развернуть проект локально
Обсуждения по переводам можно создавать в Discussions

Если захочется подсматривать/копировать части из старой документации для Vue 3, она теперь тут: https://github.com/translation-gang/docs-next

**Reply by d-darwin** [reply]
Чот я не понял, зачем столько форков насоздавали?

Если я правильно понял, то воркфлоу такой:
1. Создаем issue с название файла
2. Переводим в той же ветке (main)
3. Делаем PR

Верно?

**Reply by d-darwin** [reply]
> Чот я не понял, зачем столько форков насоздавали?

Или это все форки, включая от оригинальной репы?)

**Reply by Alex-Sokolov** [reply]
> Чот я не понял, зачем столько форков насоздавали?
> 
> Если я правильно понял, то воркфлоу такой:
> 
> 1. Создаем issue с название файла
> 2. Переводим в той же ветке (main)
> 3. Делаем PR
> 
> Верно?

Да. Все так. До рефакторинга сайта документации перевод был и тоже делался в форке

**Comment by mr-Fix** [comment]
Друзья, Вы нужны нам!
@dudintv
@sameignatovich
@MnsrIl
@Hkr32
@ivan-plotnikov
@Delari
@Askay35
@farid-temuri-lofty
@frontend-rogatnev
@Roccowen
@MaximSavin
@nzuyanov
@baraboza
@Biseamon
@3lement3

**Comment by avenikolay** [comment]
Так, я не понял, а почему все заглохло?!

**Comment by dudintv** [comment]
За себя отвечу, что работы стало много, некогда! ;) 
Тем более я уже вкладывавюсь в другие волонтерские проекты.
Наверняка у большинства откликнувшися тоже поменялось расписание и интересы.

**Reply by avenikolay** [reply]
Готов понемногу добивать это дело. Самое главное, поясните, что, куда и как.

**Reply by zavvla** [reply]
[Здесь](https://github.com/translation-gang/docs-ru) по идеи есть на русском описание. Ну или смотреть основной гайд.

**Comment by zavvla** [comment]
Подскажите, есть какие-то DOR, не хотелось бы дубли делать?

**Comment by farid-temuri** [comment]
@Jinjiang Russian translation is 99% complete, can you help us make it official?  - [HERE](https://github.com/translation-gang/docs-ru)

**Reply by Jinjiang** [reply]
@farid-temuri great job! I have sent you an invitation. Then you can feel free to transfer your repo into this org. After that, we will deploy it and make it official :-)

**Reply by Alex-Sokolov** [reply]
@Jinjiang I have rights to transfer, please send me invitation

**Reply by farid-temuri** [reply]
@Jinjiang Indeed, the owner is @Alex-Sokolov 🚀

**Reply by Jinjiang** [reply]
Sure. Invitation sent.

**Reply by Ibochkarev** [reply]
@Jinjiang Hello!

For our part, we confirmed the invite

**Reply by Ibochkarev** [reply]
@Jinjiang https://github.com/vuejs-translations/docs-ru

**Reply by Alex-Sokolov** [reply]
@Jinjiang could you also invite @Ibochkarev, he helps a lot

**Reply by Jinjiang** [reply]
invited as well

**Reply by Ibochkarev** [reply]
@Jinjiang You are now a member of Vue.js Translations!

Thanks!

**Reply by Ibochkarev** [reply]
@Jinjiang Good afternoon. 

Please increase my role and rights so that I can merge the verified pull request (as I previously did before migrating the repository)

**Reply by Jinjiang** [reply]
@Ibochkarev done. please check

**Reply by Ibochkarev** [reply]
@Jinjiang Thank you! All OK!
